turtlebot_concert/teleop.users

```diff
@@ -0,0 +1,15 @@
+###############################################################################
```

Comment:
Can we please have at least some users with different roles, so that we can see the difference in permissions when we try the demo?